api/src/api/agencies_v1/agency_routes.py

```diff
@@ -108,7 +108,7 @@ def agencies_get(db_session: db.Session, raw_list_params: dict) -> response.ApiR
     examples=examples,
 )
 @agency_blueprint.output(agency_schema.AgencySearchResponseV1Schema)
-@agency_blueprint.auth_required(api_key_auth)
+@api_key_multi_auth.login_required
```

Comment:
You added tests for the opportunity routes, could you add one for this agency route?